frontend/sozisel-app/src/components/AuthRoute/AuthRoute.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactElement, useEffect } from "react";
+import { Route, useHistory } from "react-router";
+
+import { useMeQuery } from "../../graphql";
+
+interface AuthRouteProps {
+  component: React.ComponentType<unknown> | undefined;
```

Comment:
Do not use ` | undefined` formats. Use `?` to indicate that variable may be absent.
```suggestion
  component?: React.ComponentType<unknown>;
```

Reply:
And besides that I don't like the `React.ComponentType<unknown>`. What is the point of strong typing when we randomly put this unknown thing. I'm sure react has some other types that does not require you to use generics with `unknown`.